Automattic/woocommerce-payments
# Bump qit-cli dependency to version 0.4.0

Fixes #8113 

#### Changes proposed in this Pull Request









This PR updates the `qit-cli` dependency to version `0.4.0` to enable continued successful running of QIT security scans via the `npm run test:qit` command. No additional unit tests were prepared for this change as this is changing a dependency file.

Additional context for this change can be found in this P2 post:
pcShBQ-1qu-p2

#### Testing instructions





* Check out this branch.
* The Husky post-checkout script should run `composer install` after checking out the branch, but to be sure, run: `npm run postinstall`.
* Confirm you are running version `0.4.0` with the following command: `./vendor/bin/qit --version`. You should receive the following output: `Quality Insights Toolkit CLI 0.4.0`.
* Unit tests should pass.
* Follow the 'Setup and running' instructions in the [QIT tests README file](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/qit/README.md#setup-and-running).
* The `npm run test:qit` command will run the `npm run build:release` command first, then it will run the `security.sh` script. You should see no errors and the script should exit successfully with output similar to the below example.

```
Running security tests...
Uploading zip...
 4/4 [============================] 100%
Running test...
Test run completed.
Test Run Id         <$test_run_id>                                                                                                  
Test Type           security                                                                                                 
Wordpress Version   6.4.3                                                                                                    
Woocommerce Version 8.5.2                                                                                                    
Php Version         7.4                                                                                                      
Status              success                                                                                                  
Is Development      Yes                                                                                                      
Woo Extension       WooPayments                                                                                              
Test Summary        Errors: 0 Warnings: 0                                                                                    
Version             7.1.0                                                                                                    
Result Url          https://qit.woo.com?qit_results=<$test_run_id>.<$unique_hash_string>
```

-------------------


**Post merge**
